extensions/wikia/UserLogin/css/UserLoginDropdown.globalNavigation.scss

```diff
@@ -54,7 +54,7 @@
 	}
 
 	.WikiaForm {
-		padding: 0 $large-content-padding;
+		padding: 0 $large-content-padding $large-content-padding $large-content-padding;
```

Comment:
@armonr @lizlux What do you think, would:
padding: {
	bottom: $large-content-padding;
	left: $large-content-padding;
	right: $large-content-padding;
	top: 0;
}
make things slightly clearer or it's not the best idea? :)